framework/CMakeLists.txt

```diff
@@ -85,11 +85,16 @@ if (FRAMEWORK)
     if (ENABLE_TESTING)
         find_package(CppUTest REQUIRED)
         include_directories(${CPPUTEST_INCLUDE_DIR})
+
+        find_package(Gtest REQUIRED)
+        include_directories(${GTEST_INCLUDE_DIR})
+
         add_subdirectory(tst)
     endif()
 
+set(ENABLE_TESTING ON)
+set(FRAMEWORK_TESTS ON)
 
-	celix_subproject(FRAMEWORK_TESTS "Option to build the framework tests" "OFF" DEPS)
```

Comment:
Saw your comment in #2 Re: commenting this out, but why was toggling the `"OFF"` to `"ON"` not a viable solution? Worked when I tried it earlier and it seems to be the equivalent to toggling it as a flag in the alternative command you suggested using.

Reply:
Good question! Frankly I only did that out of laziness, so there's nothing rational behind it :-/.

Are you OK with it being like that or would you prefer the other solution?

Reply:
Haha fair enough. I think it might be worth keeping the line in, but just toggling the OFF to ON. The idea of deleting lines just has me a bit worried that it might cause some issues in the future, even if I'm not sure what those issues would be. Just a personal opinion, so feel free to keep what you feel strongly about it.

Reply:
I'll make a decision later today :).

Reply:
Just merging as is. It works so good enough